### ISSUE TITLE
IDX12729: Unable to decode the header '[PII of type 'System.String' is hidden. For more details, se

### ISSUE THREAD

**Issue by reikor0x** [microsoft/Microsoft365DSC#5090]
### Description of the issue

I added the following in the export command:
"PPPowerAppsEnvironment", "TeamsAppPermissionPolicy", "TeamsAppSetupPolicy", "TeamsChannelsPolicy", "TeamsGuestMessagingConfiguration", "TeamsMeetingConfiguration", "TeamsMeetingPolicy", "TeamsMessagingPolicy", "TeamsUpgradeConfiguration", "TeamsUpgradePolicy"

After adding the following workloads I got the error below:

IDX12729: Unable to decode the header '[PII of type 'System.String' is hidden. For more details, see https://aka.ms/IdentityModel/PII.]' as Base64Url encoded string.
At [....]\Documents\WindowsPowerShell\Modules\ExchangeOnlineManagement\3.5.1\netFramework\ExchangeOnlineManagement.psm1:762 char:21
+                     throw $_.Exception.InnerException;
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (:) [], ArgumentException
    + FullyQualifiedErrorId : IDX12729: Unable to decode the header '[PII of type 'System.String' is hidden. For more details, see https://aka.ms/IdentityModel/PII.]' as Base64Url encoded string.

### Microsoft 365 DSC Version

1.24.904.1

### Which workloads are affected

Exchange Online

### The DSC configuration

```powershell
Export-M365DSCConfiguration -Components @("PPPowerAppsEnvironment", "TeamsAppPermissionPolicy", "TeamsAppSetupPolicy", "TeamsChannelsPolicy", "TeamsGuestMessagingConfiguration", "TeamsMeetingConfiguration", "TeamsMeetingPolicy", "TeamsMessagingPolicy", "TeamsUpgradeConfiguration", "TeamsUpgradePolicy","AADGroupsSettings", "O365OrgSettings", "ODSettings", "PPTenantIsolationSettings", "PPTenantSettings", "SPOTenantSettings", "TeamsMeetingBroadcastConfiguration", "TeamsMessagingPolicy", "TeamsMobilityPolicy", "TeamsOrgWideAppSettings", "EXOAcceptedDomain", "EXOActiveSyncDeviceAccessRule", "EXOAddressBookPolicy", "EXOAddressList", "EXOAntiPhishPolicy", "EXOAntiPhishRule", "EXOApplicationAccessPolicy", "EXOAtpPolicyForO365", "EXOAuthenticationPolicy", "EXOAuthenticationPolicyAssignment", "EXOAvailabilityAddressSpace", "EXOAvailabilityConfig", "EXOCalendarProcessing", "EXOCASMailboxPlan", "EXOCASMailboxSettings", "EXOClientAccessRule", "EXODataEncryptionPolicy", "EXODistributionGroup", "EXODkimSigningConfig", "EXOEmailAddressPolicy", "EXOGlobalAddressList", "EXOHostedConnectionFilterPolicy", "EXOHostedContentFilterPolicy", "EXOHostedContentFilterRule", "EXOHostedOutboundSpamFilterPolicy", "EXOHostedOutboundSpamFilterRule", "EXOInboundConnector", "EXOIntraOrganizationConnector", "EXOIRMConfiguration", "EXOJournalRule", "EXOMailboxPlan", "EXOMailContact", "EXOMailTips", "EXOMalwareFilterPolicy", "EXOMalwareFilterRule", "EXOMessageClassification", "EXOMobileDeviceMailboxPolicy", "EXOOfflineAddressBook", "EXOOMEConfiguration", "EXOOnPremisesOrganization", "EXOOrganizationConfig", "EXOOrganizationRelationship", "EXOOutboundConnector", "EXOOwaMailboxPolicy", "EXOPartnerApplication", "EXOPerimeterConfiguration", "EXOPlace", "EXOPolicyTipConfig", "EXOQuarantinePolicy", "EXORemoteDomain", "EXOReportSubmissionPolicy", "EXOReportSubmissionRule", "EXOResourceConfiguration", "EXORoleAssignmentPolicy", "EXOSafeAttachmentPolicy", "EXOSafeAttachmentRule", "EXOSafeLinksPolicy", "EXOSafeLinksRule", "EXOSharedMailbox", "EXOSharingPolicy", "EXOTransportConfig", "EXOTransportRule") -ApplicationID HIDDEN -TenantID HIDDEN -CertificateThumbprint $CertificateThumbprint -Path .\Results
```

### Environment Information + PowerShell Version

```powershell
Name                           Value
----                           -----
PSVersion                      5.1.27713.1000
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.27713.1000
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by reikor0x** [comment]
Reverting to previous workload configuration, still produces the error. It was working few days ago.

**Comment by JIO-CIPP** [comment]
You've installed an newer exchange online management module, that doesn't work yet, you should use 3.4.0 instead of 3.5.1

**Comment by reikor0x** [comment]
Fixed removing manually the 3.5.1 module file and reinstalling 3.4.0

install-Module ExchangeOnlineManagement -Scope CurrentUser -MaximumVersion 3.4.0 -force
